moon/controller.py

```diff
@@ -142,7 +142,7 @@ def send_speed_cmd(self, speed, angular_speed):
 
     def on_response(self, data):
         token, response = data
-        print(self.sim_time, "controller:response received: token=%s, response=%s" % (token, response))
+        #print(self.sim_time, "controller:response received: token=%s, response=%s" % (token, response))
```

Comment:
you could use `self.verbose` for conditional prints

Reply:
yes, I think one PR should be going over all printouts, commented out and not, and handle their on/off via a switch as you suggest. Maybe for now we should agree that all new debug prints will be implemented this way and the go back to the existing ones later.